proxy/cypress/integration/machines/list.spec.js

```diff
@@ -0,0 +1,31 @@
+import { customAlphabet } from 'nanoid'
+
+import { generateMac, login } from "../utils";
+
+const nanoid = customAlphabet('1234567890abcdefghi', 10);
+context("Machine listing", () => {
+  beforeEach(() => {
+    Cypress.config('defaultCommandTimeout', 10000);
+    login();
+    cy.setCookie('skipintro', 'true');
+    cy.visit(
+      `${Cypress.env("BASENAME")}${Cypress.env("REACT_BASENAME")}/machines`
+    );
+  });
+
+  it("renders a heading", () => {
+    cy.get("li.p-heading--four").contains("Machines");
+  });
+
+  it("can add a machine", () => {
```

Comment:
Should this be in a different file as it's about a different view?

Reply:
done